app-config.yaml

```diff
@@ -64,6 +64,10 @@ proxy:
     target: https://grafana.operate-first.cloud
     headers:
       Authorization: Bearer ${GRAFANA_TOKEN}
+  "/prometheus/api":
+    target: https://rbac-query-proxy-open-cluster-management-observability.apps.moc-infra.massopen.cloud/api/v1
+    headers:
+      Authorization: Bearer ${MCO_TOKEN}
```

Comment:
Do we have the token in vault/environment yet? Or is this TBD and that's why this PR is WIP?

Reply:
Yes we don't have it there yet so I put this PR as WIP

Reply:
I have added `MCO_TOKEN` [here](https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/secrets/k8s_secrets/show/moc/smaug/service-catalog/secrets), so now all should be setup. Is that correct? @tumido ?

Reply:
looks good :slightly_smiling_face: